src/EPR.Payment.Mopup.Function.UnitTests/Services/PaymentsServiceTests.cs

```diff
@@ -14,6 +15,7 @@
 using FluentAssertions;
 using FluentAssertions.Execution;
 using Microsoft.Extensions.Logging;
+using Microsoft.VisualBasic;
```

Comment:
is Visual Basic required? I've tried removing unnecessary using and tests still pass